lib/fathead/caniuse/redirects.txt

```diff
@@ -0,0 +1,217 @@
+pseudo elements, ::selection css pseudo element
+pseudo-elements, ::selection css pseudo element
+pseudo element, ::selection css pseudo element
+pseudo-element, ::selection css pseudo element
```

Comment:
@anjanjr "pseudo element(s)" is very vague, I don't think it should redirect to this article. Something more specific about the "selection" pseudo element would be appropriate:

- "::selection element"
- "selection element"
- "selection pseudo element"
- "css selection element"